mne/io/base.py

```diff
@@ -1492,14 +1492,16 @@ def plot(self, events=None, duration=10.0, start=0.0, n_channels=20,
              highpass=None, lowpass=None, filtorder=4, clipping=_RAW_CLIP_DEF,
              show_first_samp=False, proj=True, group_by='type',
              butterfly=False, decim='auto', noise_cov=None, event_id=None,
-             show_scrollbars=True, show_scalebars=True, verbose=None):
+             show_scrollbars=True, show_scalebars=True, time_format='float',
+             verbose=None):
         return plot_raw(self, events, duration, start, n_channels, bgcolor,
                         color, bad_color, event_color, scalings, remove_dc,
                         order, show_options, title, show, block, highpass,
-                        lowpass, filtorder, clipping, show_first_samp, proj,
-                        group_by, butterfly, decim, noise_cov=noise_cov,
+                        lowpass, filtorder, clipping, show_first_samp,
+                        proj, group_by, butterfly, decim, noise_cov=noise_cov,
```

Comment:
(nitpick) It is best to avoid cosmetic changes like this, they are distracting to PR reviewers (who must think "what changed here, is it related, is it correct?" for each change)

Reply:
I understand, that one wasn't necessary. Some other times I had to to cosmetic shifts to comply with the line limit. Or is there a better way to handle these cosmetic things?

Reply:
the other parameter-line-reflowing changes in this PR are acceptable.  However, it is possible to avoid changing too many lines simply by permitting yourself to leave a line of parameters *shorter* than it could be.  The logical conclusion of this is codified in, e.g.,  [black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) where every parameter gets its own line, always, so that exactly 1 line changes when adding 1 parameter.  

We don't currently follow black code style, but note that even if your conscience (or your BDFL) insists that you reflow parameter lines, a good practice would be to separate (in different commits) the change where the new param was added from the change where the param list was reflowed.  That allows the PR reviewer to look at changes from all commits *except* ones with messages like "whitespace" or "reflow parameter list".

Reply:
Understood, thank you.